systemd.go

```diff
@@ -74,14 +72,30 @@ func (s *SystemdController) Create(path string, resources *specs.LinuxResources)
 		newProperty("CPUAccounting", true),
 		newProperty("BlockIOAccounting", true),
 	}
-	_, err := s.conn.StartTransientUnit(name, "replace", properties, nil)
-	return err
+	ch := make(chan string)
+	_, err = conn.StartTransientUnit(name, "replace", properties, ch)
+	if err != nil {
+		return err
+	}
+	defer conn.Close()
```

Comment:
should this be moved after the `New()` call?

Reply:
yes, silly me

Reply:
fixed